packages/@uppy/core/src/index.js

```diff
@@ -909,6 +911,8 @@ class Uppy {
       throw new Error(msg)
     }
 
+    this.log(`Using ${pluginId} v${Plugin.VERSION}`)
```

Comment:
might want to check if it exists, most userland plugins won't have it for example

Reply:
Added a check, thanks!